client/logic/game/game.gd

```diff
@@ -1,23 +1,19 @@
-extends Camera2D
+extends Node
 
 signal round_change_ended
 @export var _shop_scene: PackedScene
 @export var _move_layer: PackedScene
 @export var _move_arrow: PackedScene
 @export var _floating_info: PackedScene
 
-@onready var _selection_decal: Sprite2D = $"../SelectionDecal"
-@onready var _action_panel: ActionPanel = $"../../../Interface/ActionPanel"
-@onready var _input_timer: Timer = $"../InputTimer"
+@onready var _action_panel: ActionPanel = $"../../Interface/ActionPanel"
 @onready var _round_button: Button = %RoundButton
-@onready var _escape_panel: Control = %EscapePanel
-@onready var _map_slot: Node = %MapSlot
-@onready var _cursor: Cursor = %Cursor
 @onready var _round_label: RoundLabel = %RoundLabel
 @onready var _round_rect: Polygon2D = %RoundRect
 @onready var _round_number_label: Label = %RoundNumberLabel
 @onready var _money_label: Label = %MoneyLabel
 @onready var _animation_player: AnimationPlayer = %AnimationPlayer
+@onready var _input: GameInput = $"../GameInput"
```

Comment:
not a big fan of the ../ maybe we can use the export, and assign it into the scene?